config.py

```diff
@@ -47,6 +47,11 @@ class vedaAppSettings(BaseSettings):
         description="Custom domain name, i.e. veda-backend.xyz",
     )
 
+    veda_bootstrap_qualifier: Optional[str] = Field(
+        None,
+        description="Qualifier for cdk bootstraps",
```

Comment:
Custom bootstrap qualifier override if not using a default installation of AWS CDK Toolkit to synthesize app.